plugins/processors/aws/ec2/ec2_sample_config.go

```diff
@@ -4,5 +4,52 @@
 package ec2
```

Comment:
shouldn't this file go away?

Reply:
Good catch! This plugin is down another directory level, compared to all other plugins. That's why my automatic conversion script didn't catch it.
Anyway, along the way I also found that generation is not triggered due to a missing `s` in the `Makefile`... *facepalm*

Both fixed...